libs/content/workspace/src/storage_viewer/data.rs

```diff
@@ -0,0 +1,162 @@
+use lb_rs::Uuid;
+use lb_rs::{blocking::Lb, model::file::File};
+use serde::Deserialize;
+use std::collections::HashMap;
+
+#[derive(Debug)]
+pub struct Data {
```

Comment:
You should use visibility to your advantage, most of these things don't need to be `pub`.